packages/xo-server-audit/src/index.js

```diff
@@ -335,16 +335,18 @@ class AuditXoPlugin {
       }
 
       // check the integrity of all stored hashes
-      integrityCheckSuccess = await Promise.all(
-        hashes.map((oldest, key) =>
-          oldest !== lastHash
-            ? this._checkIntegrity({ oldest, newest: hashes[key + 1] })
-            : true
-        )
-      ).then(
-        () => true,
-        () => false
-      )
+      try {
+        for (let i = 0; i < hashes.length - 1; ++i) {
+          await this._checkIntegrity({
+            oldest: hashes[i],
+            newest: hashes[i + 1],
+          })
+        }
+
+        integrityCheckSuccess = true
+      } catch (_) {
```

Comment:
@julien-f Should i use https://github.com/tc39/proposal-optional-catch-binding ? It was released in `Babel 7`

Reply:
No, I don't think the small improvement of this syntax is worth adding a constraint to the build system.